packages/expo-maps/ios/ExpoMaps/AppleMaps/AppleMapsDelegate.swift

```diff
@@ -2,14 +2,12 @@ import MapKit
 
 class AppleMapsDelegate: NSObject, MKMapViewDelegate {
   private let markersManager: AppleMapsMarkersManager
-  private let sendEvent: (String, [String: Any?]) -> Void
```

Comment:
this doesnt seem to be used anywhere in the delegate or elsewhere in the module's code, and there were no compile-time errors, so I figured it's okay to remove this.